docs/directives.md

```diff
@@ -508,3 +509,19 @@ Returns a URL to the generated image. This is the default when your directives p
 ```js
 import Image from 'example.jpg?w=500' // the type of Image is a string and it's a URL to the transformed image
 ```
+
+---
+
+### Inline
+
+• **Keyword**: `inline`<br> • **Type**: _boolean_<br>
+
+Return base64 encoded inline image instead of URL to the generated image. This can be combined with some output directives like `url`, `srcset` or `picture`.
```

Comment:
it's pretty unclear what should happen if you use both `url` and `inline` and I'm not sure I'd recommend those two be used together
```suggestion
Return base64 encoded inline image instead of URL to the generated image. This can be combined with some output directives like `srcset` or `picture`.
```

Reply:
Updated